src/documentation/common.md

```diff
@@ -58,4 +58,75 @@ The files `contributors.txt` and `datasets.txt` provide additional information a
 | datasets.txt | dataset_id | dataset_id | Required | This field is prefixed.
 | datasets.txt | contributor_id | contributor_id | Required | This field is prefixed.
 | datasets.txt | dataset_start_date |  |  | Smallest date of all the trips of the dataset.
-| datasets.txt | dataset_end_date |  |  | Greatest date of all the trips of the dataset.
\ No newline at end of file
+| datasets.txt | dataset_end_date |  |  | Greatest date of all the trips of the dataset.
+
+## Common practices
+The following rules apply to every converter, unless otherwise explicitly specified.
+- When one or more stop_points in the input data are not attached to a stop_area, 
+a stop_area is automatically created. The coordinates of the new stop_area are 
+computed as the barycenter of the associated stop_points. (The same rule applies 
+in case the stop_area exists in the input data, but the coordinates are not specified.)
+- Unless otherwise specified, dates of service are trasnformed into a list of active dates as if using a single NTFS file `calendar_dates.txt`.
+
+| NTFS file | NTFS field | Constraint | Note |
+| --- | --- | --- | --- |
+| calendar_dates.txt | service_id | Required | All slashes `/` are removed
+| calendar_dates.txt | date | Required | 
+| calendar_dates.txt | exception_type | Required | Fixed value `1`.
+
+## Sanitizer
+The sanitizer checks for incoherences in the model and also cleans up all dangling
```

Comment:
the function se call sanitize does not check incoherence. incoherences (2 same id) are check when a collection of objects is created

Reply:
My bad, that's on me, @papailio just copied over the previous text I created in `sanitizer.md`... and you're right, no coherence check is done, only removing objects that are not referenced.

Reply:
What about changing the name "sanitizer" with "Consistancy" ? this spec is not about where the checks are made, but what controls are effective.